packages/ruby/lib/readme/har_request.rb

```diff
@@ -1,18 +1,18 @@
 module Readme
   class HarRequest
-    def initialize(request, filter_params = [])
+    def initialize(request, filter = Filter::None.new)
```

Comment:
Is the default necessary here? I think we always return `Filter::None.new` when calculating a filter in the first place?